src/gui/KM_InterfaceGamePlay.pas

```diff
@@ -2649,7 +2653,7 @@ procedure TKMGamePlayInterface.AlliesOnPlayerSetup(Sender: TObject);
         and gGame.Networking.NetPlayers[NetI].IsHuman then // and is not Computer
       begin
         Update_Image_AlliesMute(Image_AlliesMute[I]);
-        Image_AlliesMute[I].Show;
+        Image_AlliesMute[I].Visible := True;

```

Comment:
well they should, but they are not ^) Show will *really* show control - it checks if parent is visible or not, if not, then show him too etc. Just settig visible will not change parent visible status.
This is not related to this PR actually.